src/Middle.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes,Route, Link} from "react-router-dom";
+import App from './App';
+import Home from './Home';
+
+
+function Middle() {
+    return (
+        <div>
+            <Router>
```

Comment:
why you use middle.jsx and mediator.jsx ?

Reply:
+1